test/unit/dataMap/metaManager/metaManager.spec.js

```diff
@@ -0,0 +1,236 @@
+    });
+  });
+
+  describe('getCellsMetaAtRow()', () => {
+    it('should pass a method call to CellMeta layer', () => {
+      const metaManager = new MetaManager();
+
+      spyOn(metaManager.cellMeta, 'getMetasAtRow').and.returnValue(['foo']);
+
+      expect(metaManager.getCellsMetaAtRow(32)).toEqual(['foo']);
+      expect(metaManager.cellMeta.getMetasAtRow).toHaveBeenCalledWith(32);
+    });
+  });
+
+  describe('createRow()', () => {
+    it('should pass a method call to only CellMeta layer', () => {
+      const metaManager = new MetaManager();
+
+      spyOn(metaManager.cellMeta, 'createRow');
+
+      expect(metaManager.createRow(32)).toBeUndefined();
+      expect(metaManager.cellMeta.createRow).toHaveBeenCalledWith(32, 1);
+
+      expect(metaManager.createRow(22, 10)).toBeUndefined();
+      expect(metaManager.cellMeta.createRow).toHaveBeenCalledWith(22, 10);
+    });
+  });
+
+  describe('removeRow()', () => {
+    it('should pass a method call to only CellMeta layer', () => {
+      const metaManager = new MetaManager();
+
+      spyOn(metaManager.cellMeta, 'removeRow');
+
+      expect(metaManager.removeRow(32)).toBeUndefined();
+      expect(metaManager.cellMeta.removeRow).toHaveBeenCalledWith(32, 1);
+
+      expect(metaManager.removeRow(22, 10)).toBeUndefined();
+      expect(metaManager.cellMeta.removeRow).toHaveBeenCalledWith(22, 10);
+    });
+  });
+
+  describe('createColumn()', () => {
+    it('should pass a method call to CellMeta and ColumnMeta layers', () => {
+      const metaManager = new MetaManager();
+
+      spyOn(metaManager.cellMeta, 'createColumn');
+      spyOn(metaManager.columnMeta, 'createColumn');
+
+      expect(metaManager.createColumn(32)).toBeUndefined();
+      expect(metaManager.cellMeta.createColumn).toHaveBeenCalledWith(32, 1);
+      expect(metaManager.columnMeta.createColumn).toHaveBeenCalledWith(32, 1);
+
+      expect(metaManager.createColumn(22, 10)).toBeUndefined();
+      expect(metaManager.cellMeta.createColumn).toHaveBeenCalledWith(22, 10);
+      expect(metaManager.columnMeta.createColumn).toHaveBeenCalledWith(22, 10);
+    });
+  });
+
+  describe('removeColumn()', () => {
+    it('should pass a method call to CellMeta and ColumnMeta layers', () => {
+      const metaManager = new MetaManager();
+
+      spyOn(metaManager.cellMeta, 'removeColumn');
+      spyOn(metaManager.columnMeta, 'removeColumn');
+
+      expect(metaManager.removeColumn(32)).toBeUndefined();
+      expect(metaManager.cellMeta.removeColumn).toHaveBeenCalledWith(32, 1);
+      expect(metaManager.columnMeta.removeColumn).toHaveBeenCalledWith(32, 1);
+
+      expect(metaManager.removeColumn(22, 10)).toBeUndefined();
+      expect(metaManager.cellMeta.removeColumn).toHaveBeenCalledWith(22, 10);
+      expect(metaManager.columnMeta.removeColumn).toHaveBeenCalledWith(22, 10);
+    });
+  });
+
+  describe('clearCellsCache()', () => {
+    it('should pass a method call to only CellMeta layer', () => {
```

Comment:
This test checks if the `cellMeta.clearCache` was called. But does not verify if it was the **only** one. None other method is spied on, so how do we know? The description may be misleading here

Reply:
Fixed within https://github.com/handsontable/handsontable/commit/cf956e58ac29d444754574da8a6b73a1c302e1a2.